pkg/syncer/cnsoperator/manager/init.go

```diff
@@ -93,9 +93,10 @@ func InitCnsOperator(ctx context.Context, clusterFlavor cnstypes.CnsClusterFlavo
 				return err
 			}
 		}
-
+		multivCenterCSITopologyEnabled := commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx,
+			common.MultiVCenterCSITopology)
 		volumeManager, err = volumes.GetManager(ctx, vCenter, nil,
-			false, false, false, commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.ListViewPerf))
```

Comment:
cnsoperator is for wcp. we don't need to check for this fss in wcp.

Reply:
reverted fss check